### DISCUSSION TITLE
Access to font from origin has been blocked by CORS policy

### DISCUSSION THREAD

**Discussion by harishmathur** [webpack/webpack#18122] [Q&A]
I am implementing Webpack Module Federation with 1 Host app and 1 child app.

When I hit the Host app URL, I get below CORS error (even though I have enabled CORS on both Host and Child app and also allowing myorg.net.

Access to font at 'https://mychild-app.apps.myorg.net/static/media/open-sans-latin-400-normal.xxxxxxxxx.woff2' from origin 'https://myhost-app.apps.myorg.net' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

Please let me know if anything similar is a known issue with Webpack Module Federation

**Comment by alexander-akait** [comment]
You need to setup CORS headers for your servers, there are no problems with webpack here
